src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -644,7 +656,7 @@ export default function AppointmentsPage() {
                 key={status}
                 status={status}
                 slot={slot?.id}
-                practitioner={practitioner?.id ?? null}
+                practitioners={qParams.practitioners ?? null}
```

Comment:
```suggestion
                practitioners={qParams.practitioners || null}
```

handles the empty string case